src/pages/FlagCommentPage.js

```diff
@@ -200,10 +199,4 @@ FlagCommentPage.displayName = 'FlagCommentPage';
 export default compose(
     withLocalize,
     withReportAndReportActionOrNotFound,
-    withOnyx({
```

Comment:
This can be removed because the `parentReportAction` comes from `withReportAndReportActionOrNotFound` now.